docs/QUICK-START.md

```diff
@@ -134,6 +134,7 @@ NSC nsc-84799b768f-wvxwt with IP 10.30.1.1/30 pinging vppagent-icmp-responder-ns
 Validating the VPN example will output the following:
 
 ```bash
+./scripts/verify_vpn_gateway.sh
```

Comment:
Actually this script is called by `make k8s-check`, I don't think we need it here.